index.js

```diff
@@ -8,83 +9,106 @@ const maxGuessesMessage = document.getElementById('max-guesses');
 const numberOfGuessesMessage = document.getElementById('number-of-guesses');
 const correctMessage = document.getElementById('correct');
 
-let targetNumber;
-let attempts = 0;
-const maxNumberOfAttempts = 5;
+// Initialize variables
+let targetNumber; // The random number to guess
+let attempts = 0; // Number of attempts made by the user
+const maxNumberOfAttempts = 5; // Maximum number of attempts allowed
 
-// Returns a random number from min (inclusive) to max (exclusive)
-// Usage:
-// > getRandomNumber(1, 50)
-// <- 32
-// > getRandomNumber(1, 50)
-// <- 11
+// Function to generate a random number between min (inclusive) and max (exclusive)
 function getRandomNumber(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
+// Function to handle user's guess
 function checkGuess() {
-  // Get value from guess input element
+  // Get the user's guess from the input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  attempts++; // Increment the number of attempts
 
+  // Hide all message elements initially
   hideAllMessages();
 
+  // Check if the guess matches the target number
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+    // Display the number of guesses made
+    numberOfGuessesMessage.style.display = 'block';
+    numberOfGuessesMessage.textContent = `You made ${attempts} guesses`;
```

Comment:
A small observation: it would be great to also change "guesses" to "guess" here when there was only one attempt, as you do below on line 57.